sdk/javascript/__tests__/v1/client_test.js

```diff
@@ -86,10 +88,10 @@ describe('Client', () => {
         return test_records;
     }
 
-    const client = new Client(new Config());
+    const client = new Client(Config.loadFromEnv());
 
     beforeEach(async () => {
-        // await setTimeout(3000); // Wait for api server
+        await setTimeout(3000); // Wait for api server
```

Comment:
note: this may become flaky. better check would be to curl the endpoint with a timeout until it becomes available. we dont need to do this now, just want to bring awareness if the issue arrises in the future